hugo/content/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: About
+
+---
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The source code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip/).
+
+## Licenses
+
+Mumble is Free and Open-Source Software.
+The core project is licensed under a BSD-style license. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
+
+Our other projects may use different licenses. For details we refer to the respective GitHub repositories, see: [Overview of all Mumble projects on GitHub](https://github.com/mumble-voip/).
```

Comment:
Is this a typo or intended this way?

```suggestion
Our other projects may use different licenses. For details please refer to the respective GitHub repositories, see: [Overview of all Mumble projects on GitHub](https://github.com/mumble-voip/).
```

Reply:
It was intended this way, but I think your suggestions is better

Reply:
Agreed.